ui/app/build-configs/directives/pnc-select-build-parameters/pncEnterBuildParameters.js

```diff
@@ -40,6 +41,9 @@
 
     // --------------------
 
+    $ctrl.updateEnumParameterValues = () => {
+      $ctrl.enumParameterValues = $ctrl.enumParameters.filter(parameter => parameter.name === $ctrl.key)?.[0]?.enumValues;
+    };
```

Comment:
Hi @matedo1 ,
I think we need to change our rules for code format since this part doesn't pass the jshint validation: 
![image](https://user-images.githubusercontent.com/6870455/112391649-25d9c300-8cf0-11eb-85fa-36223602ae40.png)

Reply:
Hi @DnsZhou then feel free to keep the original version.